python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -93,8 +97,10 @@ def start_ganga():
     logger.info("Checking Credentials")
 
     if missing_cred:
-        raise Exception("Failed due to missing credentials %s" %
-                        str(missing_cred))
+        raise Exception("Failed due to missing credentials %s" % str(missing_cred))
+
+    from Ganga.GPI import queues
+    queues.unlock()
```

Comment:
Yes I agree, I think this would be much nicer to have but I think this is work related to startup/shutdown re-working and breaking up of bootstrap that's going on :)